.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
.idea raczej do gitignore

Reply:
Właśnie ja w IDE tego nei widzę nie czy to nie jest dla wszystkich userów jetbrains wspólny config po prostu xD